content/developer/reference/frontend/javascript_reference.rst

```diff
@@ -1432,55 +1432,72 @@ char (FieldChar)
   - Supported field types: *char*
 
 date (FieldDate)
-  This is the default field type for fields of type *date*. Note that it also
-  works with datetime fields.  It uses the session timezone when formatting
-  dates.
+  This is the default field type for fields of type *date*. It consists of a text
+  box and a date picker.
 
-  - Supported field types: *date*, *datetime*
+  - Supported field types: *date*
 
   Options:
 
-  - datepicker: extra settings for the datepicker_ widget.
+  - min_date / max_date: sets limit dates for accepted values. By default, the earliest
+    accepted date is *1000-01-01* and the latest is *9999-12-31*.
+    Accepted values are: SQL-formatted dates (yyyy-MM-dd HH:mm:ss) or *today*.
+
+  .. code-block:: xml
+
+      <field name="datefield" options="{'min_date': 'today', 'max_date': '2023-12-31'}" />
```

Comment:
Indent this code-block and the second one below to make them part of the bullet points.
```suggestion
    .. code-block:: xml

        <field name="datefield" options="{'min_date': 'today', 'max_date': '2023-12-31'}" />
```